HelpSource/Classes/Routine.schelp

```diff
@@ -162,9 +163,7 @@ See also: link::Classes/Object#-yield::, link::Classes/Object#-alwaysYield::
 
 method::play
 
-In the SuperCollider application, a Routine can be played using a link::Classes/Clock::, as can any link::Classes/Stream::.
-every time the Routine yields, it should do so with a float, the clock will interpret that, usually
-pausing for that many seconds, and then resume the routine, passing it the clock's current time.
+Schedules the Routine on the given link::Classes/Clock::, at a time specified by code::quant::. At that time, the routine will "awake," setting the clock and time and evaluating the routine. If the routine yields a number, this number of beats will be added to the current time and the routine will be rescheduled. (This behavior is compatible with scheduling a link::Classes/Stream:: or a link::Classes/Function::.)
```

Comment:
scare quotes unnecessary, parens unnecessary.

'wake up' is more standard terminology. Also would recommend capitalizing Routine throughout this paragraph